firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -700,6 +90,10 @@ void setup( )
 
     init_can( );
 
+	pid_params.derivative_gain = current_ctrl_state.SA_Kd;
```

Comment:
Probably better to change these to use a macro instead of a variable from the control state structure.